femitj/iReporter
# #162495476 Create log-in Endpoint for User

**What does this PR do?**

Creates a log-in endpoint for User account

**Description of the task to be completed**

Ensure the following endpoints are working as they should

POST/api/v1/auth/login

**How should this be tested manually**

* After you clone the repo, 
* cd into it and run npm install to install node modules. 
* Make sure you have nodejs and express installed. 
* Afterwards, run npm start to initiate the app.

test endpoint above using postman with localhost:8080 as the host
log-in with: email e.g something@mail.com,
                      password,

**What are the relevant pivotal stories**
[#162495476](https://www.pivotaltracker.com/story/show/162495476)
